aten/src/ATen/cuda/tunable/Tunable.cpp

```diff
@@ -364,6 +367,22 @@ bool TuningContext::IsTuningEnabled() const {
   return tuning_enable_;
 }
 
+void TuningContext::EnableNumericsCheck() {
+  numerics_check_enabled_ = true;
+}
+
+void TuningContext::DisableNumericsCheck() {
```

Comment:
Maybe combine the two APIs into One so that we have one Set API and one Get API?

Reply:
Good idea.  Done.